tests/foreman/api/test_syncplan.py

```diff
@@ -258,9 +295,19 @@ def test_update_sync_plan_with_random_interval(self, interval):
         @Feature: SyncPlan
 
         """
+        initial_interval = [
```

Comment:
And it is much better, as we're avoiding predefined values. Thanks for suggestion :+1: